webclient/src/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    
+    <meta name="from-view" content="<!-- @echo view -->">
```

Comment:
I don't mind putting this there, just wondering whether there is a reason for this.

Reply:
Reasoning being, that if at some point the ngnix conifig acts up we might notice quicker where to look :)
Is not nessesary, but I thought this might be a nice to have